src/chains/celo/serializers.test.ts

```diff
@@ -443,15 +442,77 @@ describe('invalid params specific to CIP-64', () => {
   })
 })
 
-describe.each([
-  { typeName: 'CIP-42', baseTransaction: baseCip42 },
-  { typeName: 'CIP-64', baseTransaction: baseCip64 },
-])('Common invalid params (for $typeName)', ({ typeName, baseTransaction }) => {
+describe('Common invalid params (for CIP-42)', () => {
+  test('invalid to', () => {
+    const transaction: TransactionSerializableCIP42 = {
+      ...baseCip42,
+      to: '0xdeadbeef',
+    }
+    expect(() => serializeTransactionCelo(transaction)).toThrowError(
+      InvalidAddressError,
+    )
+  })
+
+  test('gatewayFeeRecipient is not an address', () => {
+    const transaction: TransactionSerializableCIP42 = {
+      ...baseCip42,
+      // @ts-expect-error - (Type '"example"' is not assignable to type "`0x${string}"'
+      gatewayFeeRecipient: 'example',
+    }
+    expect(() => serializeTransactionCelo(transaction)).toThrowError(
+      TipAboveFeeCapError,
```

Comment:
this doesnt seem like its the right error. I think this throw is unrelated to the gatewayFee field

Reply:
correct